third_party/xla_client/BUILD

```diff
@@ -193,7 +194,6 @@ cc_library(
     "@org_tensorflow//tensorflow/compiler/xrt/cc:xrt_ops",
     "@org_tensorflow//tensorflow/core:core_cpu",
     "@org_tensorflow//tensorflow/core:framework_internal",
-    "@org_tensorflow//tensorflow/core:lib",
```

Comment:
You don't need to delete this reference or the one under `xrt_local_service`. These are only used by XRT.

Reply:
Thanks, Will, updated